app/src/main/java/at/roteskreuz/stopcorona/model/repositories/HandshakeCodewordRepository.kt

```diff
@@ -57,4 +58,18 @@ class HandshakeCodewordRepositoryImpl(
     private fun getResourceId(identifier: String): Int {
         return contextInteractor.resources.getIdentifier("handshake_code_$identifier", "string", contextInteractor.packageName)
     }
+
+    override fun zeroPrefixed(number: Int): String {
+        val value = abs(number)
+        if (value < RANDOM_IDENTIFICATION_MIN ||
+            value > RANDOM_IDENTIFICATION_MAX ||
+            number < 0) {
+            Timber.e(SilentError("Invalid input number $number"))
```

Comment:
what are you doing if the number is incorrect? you still go on with it, right?

Reply:
yes, what do you suggest to do instead?
`If there is number greater than 9999, the result is last 4 characters of the string value.`